integration_test.go

```diff
@@ -261,34 +263,34 @@ func integrationtestHelperCartographer(t *testing.T, subAlgo viamcartographer.Su
 		// save the internal state of the mapping run to a new datadir
 		saveInternalState(t, internalState, dataDirectoryLocalizing)
 		// localize on that internal state
-		testHelperCartographer(t, dataDirectoryLocalizing, subAlgo, logger, false, true, 0, cartofacade.LocalizingMode)
+		testHelperCartographer(t, dataDirectoryLocalizing, subAlgo, logger, true, false, 0, cartofacade.LocalizingMode)
 	})
 
-	t.Run("replay sensor localizing mode", func(t *testing.T) {
```

Comment:
[q] did these tests switch orders? I see a similar change on line 245

Reply:
No they did not, GitHub marking these lines might have to do with changing the use imu / don't use imu test ordering

Reply:
Probably the linter more than github